lib/LANraragi/Utils/Routing.pm

```diff
@@ -86,6 +86,8 @@ sub apply_routes {
     $logged_in->get('/logs/mojo')->to('logging#print_mojo');
     $logged_in->get('/logs/redis')->to('logging#print_redis');
 
+    $public_routes->get('/rgroups')->to('readinggroup#index');
```

Comment:
Which actually takes me a question I had when working on this: Should groups be user specific? I mean, suppose there are 3 persons using the platform, Should the 3 of them be looking at all the groups, or should everyone look only the lists they created? I think I'll add a public/private attribute anyway, just in case

Reply:
Well, technically there's only two possible users atm, logged in/admin or not logged in. I believe groups should be like categories and be accessible to all.. 

I've been thinking about adding a secondary password for reading/uploading for https://github.com/Difegue/LANraragi/issues/630 and https://github.com/Difegue/LANraragi/issues/740, but account-specific content is a bit out of that scope 🤔